### DISCUSSION TITLE
repo showing "Dangerous" why

### DISCUSSION THREAD

**Discussion by Akash-Gupta-git** [recodehive/Stackoverflow-Analysis#421] [General]
@sanjay-kv  sir I am facing problem by my repository  I have created some repository. I can see output on web by clicking that link. and starting people and I can see the repository publicly. but now it is not working. It is showing me "Dangerous"
![image](https://github.com/user-attachments/assets/d8e5e3f3-e726-4be7-95ed-726a2fb24dcc)

after clicking this 
![image](https://github.com/user-attachments/assets/afd1703a-c86c-4a1d-b09d-e43b5bc3db75)
please help me 🙏
 please Solve this problem

**Comment by sanjay-kv** [comment]
i cant solve this probel, give me the repo link here. 

If you deal something with currency and altering it, github pages automatically flag it

**Reply by Akash-Gupta-git** [reply]
No, all my repos show this. And I don't deal with currency you can check my any link like  
https://akash-gupta-git.github.io/Akash-Cube-Weather/
https://akash-gupta-git.github.io/Netflix-TV/
https://akash-gupta-git.github.io/Amazon-Clone/
https://akash-gupta-git.github.io/akash-portfolio/

these all have the same problem "Dangerous" why 
if you want to view on web then we have to click on >>details<< button option then >>this unsafe site<< when showing dangerous

**Comment by sanjay-kv** [comment]
@hemu21 have you encountered anything like this?

**Comment by hemanthkumar-21** [comment]
@sanjay-kv  I doesn't face this any time, but one of my friend face the face issue.

@Akash-Gupta-git I can suggest you few tips once try those. I am not sure they will work but once try They may work. I hope this will help you.

### STEP - 1

![image](https://github.com/user-attachments/assets/ed26265a-2f95-41c8-93ac-b9c6ad25aa25)

Make sure you had enabled it.  Try this once. If it doesn't work go to step 2

### STEP - 2

Once create the new repo with another name and try to deploy it. If it doesn't work go to step 3

### STEP - 3

Once try to change your GitHub username to not have any `-` or special characters. once try using normal username like caps, normal, numerical without special characters. 

### STEP - 4

If above any doesn't work. I suggest you use vercel or Render. If you want to use GitHub pages only then try creating new GitHub account.
